python/pyspark/ml/regression.py

```diff
@@ -619,6 +627,22 @@ def getSubsamplingRate(self):
         """
         return self.getOrDefault(self.subsamplingRate)
 
+    @since("1.4.0")
+    def setFeatureSubsetStrategy(self, value):
+        """
+        Sets the value of :py:attr:`featureSubsetStrategy`.
+
+        .. note:: Deprecated in 2.1.0 and will be removed in 2.4.0.
```

Comment:
sorry, this should be `.. note:: Deprecated in 2.4.0 and will be removed in 3.0.0.`

Reply:
Sorry.  Fixed.